configuration/configuration.go

```diff
@@ -123,6 +124,22 @@ type Configuration struct {
 	} `yaml:"redis,omitempty"`
 }
 
+// Handlers is for handling error configuration, when some kinds of error happens.
+// Catch the error, and send a warning mail.
+type Handlers struct {
+	Errors struct {
+		Type     string   `yaml:"type,omitempty"`
+		Host     string   `yaml:"smtp_host,omitempty"`
+		Port     string   `yaml:"smtp_port,omitempty"`
+		Login    string   `yaml:"smtp_login,omitempty"`
+		Password string   `yaml:"smtp_password,omitempty"`
+		Insecure bool     `yaml:"smtp_insecure,omitempty"`
+		FromAddr string   `yaml:"from_addr,omitempty"`
+		ToAddr   []string `yaml:"to_addr,omitempty"`
+		Level    string   `yaml:"level,omitempty"`
```

Comment:
Use the `LogLevel` type for this field so it automatically parses and validates the value.